blankiety_galantow/resources/game/js/main.js

```diff
@@ -75,7 +75,7 @@ const app = new Vue({
             // First click reveals the stack
             stack.revealed = true;
 
-            let card_ids = [];
+            let card_iupdateTimerds = [];
```

Comment:
WTF is this?

Reply:
OMG, chyba jakiś skrót klawiszowy mnie zdradził... już poprawiłam jakby co